cli/src/test/kotlin/AuthLogoutCommandTest.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import com.github.ajalt.clikt.command.test
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.should
+import io.kotest.matchers.shouldBe
+
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockkObject
+import io.mockk.runs
+import io.mockk.unmockkAll
+import io.mockk.verify
+
+import org.eclipse.apoapsis.ortserver.cli.OrtServerMain
+import org.eclipse.apoapsis.ortserver.cli.model.AuthenticationStorage
+
+class AuthLogoutCommandTest : StringSpec({
+    afterEach { unmockkAll() }
+
+    "logout command" should {
+        "delete the authentication file" {
+            mockkObject(AuthenticationStorage)
```

Comment:
Why not just use the `AuthenticationStorage` instead of mocking it?

Reply:
Because I don't want to overwrite the actual `auth.yml` file on the machine of the developer executing the test.

But you're right I shouldn't mock the object under test.

Reply:
Ok, I guess I just don't like that we need mocking to test a trivial function.